tests/test_field_for_schema.py

```diff
@@ -89,13 +89,11 @@ class Color(Enum):
         )
 
     def test_union(self):
-        import marshmallow_union
+        import marshmallow_polyfield
 
         self.assertFieldsEqual(
             field_for_schema(Union[int, str]),
-            marshmallow_union.Union(
-                fields=[fields.Integer(), fields.String()], required=True
-            ),
+            marshmallow_polyfield.PolyField(required=True),
```

Comment:
This test is not covering the same checks, we should be checking the content of PolyField somehow

Reply:
not easy, PolyField is just a pointer on 2 functions (serialize and deserialize)

Reply:
:ah: ok I see, well let's keep it as is then